week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -38,7 +64,13 @@ Write a function that:
 */
 
 function formatPercentage(arr) {
-  
+  var newArr = [];
+  for (let i = 0; i < arr.length; i++) { // Loop for reed items in aryy 
+    let greaterThan100 = (arr[i] > 100); // If number greater than 100 return true if not return false
+    let checkdecimalNumber = arr[i] % 1 == 0; // If number hase decimal (0.372) return true if not return false
+    newArr.push(`${(greaterThan100? 100 : checkdecimalNumber? arr[i]:arr[i].toFixed(2))}%`); // if number greater than 100 return 100, if hase decimal fixed to two decimal number if not return number
```

Comment:
I think it's better to break this up, chaining two ternary operators makes this harder to read for anyone else.